pop-control.lua

```diff
@@ -0,0 +1,124 @@
+-- Script to control the various population caps as well as use of max-wave and hermit persistently per fortress
```

Comment:
This is too long for `ls`. How about just "control the various population caps"?